pode.build.ps1

```diff
@@ -655,7 +655,7 @@ Task Install-Module -If ($Version) Pack, {
     $path = './pkg'
 
     # copy over folders
-    $folders = @('Private', 'Public', 'Misc', 'Libs', 'licenses')
```

Comment:
You'll need to update the `/packers/choco/tools/ChocolateyInstall_template.ps1` as well, so it knows to copy over the `/Locales` folder :)